### ISSUE TITLE
[1.20.4] Latest Figura + Fabric for titular version causes a crash when exiting fullscreen with Windows key or Alt+Tab while connected to a server.

### ISSUE THREAD

**Issue by Cris2027** [FiguraMC/Figura#269]
### Bug Description

While tabbed out of Minecraft Java to a secondary app such as a browser or a social app results in the game crashing if the player was connected to a multiplayer server before tabbing out via either the Windows key or Alt+Tab keys.

Resulting error code: -1073740791
Known causes of this error according to google searches: outdated graphic drivers, lack of alocated RAM, secondary app conflicts.

Methods tried: updating drivers, alocating more RAM, closing every secondary app before launching, updating both Fabric and Figura to latest versions.
Results: same error with same procedure, varied timing.

Hope you can help with this error soon and thanks either way!

### Reproduction Steps

Use Fabric-0.16.9-1.20.4 as the launcher version on the official launcher.

In mods folder use only Figura-0.1.5-rc.3+1.20.4.

Once loaded join any multiplayer server (for me didn't matter which).

Once inside, Tab out of minecraft to any background app (Discord, Chrome, Opera...).

Error should manifest itself between 20s and 10min after tabbing out..

### Log File

[latest.log](https://github.com/user-attachments/files/17652536/latest.log)

**Comment by Riftlight** [comment]
Can you provide a crash report log (found in the crash-reports folder)? I doubt that this is a figura issue.

**Comment by Cris2027** [comment]
I can't, I don't know why it doesn't generate one, best I can give you is the Java environment log,
Sorry about that and if it's not too much to ask if you could help me it would be amazing, thanks regardless

**Comment by PoolloverNathan** [comment]
> I can't, I don't know why it doesn't generate one, best I can give you is the Java environment log, [https://mclo.gs/Uhs1LkI](https://mclo.gs/Uhs1LkI)
> Sorry about that and if it's not too much to ask if you could help me it would be amazing, thanks regardless

Publishing this type of file is a very bad idea — please refresh your Minecraft session token immediately.

**Comment by Cris2027** [comment]
> Publishing this type of file is a very bad idea — please refresh your Minecraft session token immediately.

My bad, done. Appart from that, is there anything I can do to fix it?

**Comment by PoolloverNathan** [comment]
> 
> > Publishing this type of file is a very bad idea — please refresh your Minecraft session token immediately.
> 
> My bad, done. Appart from that, is there anything I can do to fix it?

I remember this being on the Minecraft jira, so not sure. Probably just general Windows jank.

**Comment by Cris2027** [comment]
> I remember this being on the Minecraft jira, so not sure. Probably just general Windows jank.

So I guess apart from almost doxing myself there's nothing else to do here?

**Comment by PoolloverNathan** [comment]
> > I remember this being on the Minecraft jira, so not sure. Probably just general Windows jank.
> 
> So I guess apart from almost doxing myself there's nothing else to do here?

Not really. Have you tried making sure that a little part of Minecraft is always visible? It might be getting deprioritized due to being invisible,  eventually causing it to get killed.

**Comment by Cris2027** [comment]
I'll have to try it, thanks

**Comment by Cris2027** [comment]
> Not really. Have you tried making sure that a little part of Minecraft is always visible? It might be getting deprioritized due to being invisible, eventually causing it to get killed.

So update: it kinda fixes it, thanks. I still need to figure out why it's happening and fix the bigger issue since it doesn't explain why it happens when Figura is installed but so far, thanks for the help
